peekingduck/configs/draw/legend.yml

```diff
@@ -1,6 +1,6 @@
 input: ["all"]
 output: ["img"]
 
-all_legend_items: ["fps", "count"]
+all_legend_items: ["fps", "count", "zone_count"]
 position: "bottom"
 include: ["all_legend_items"]
```

Comment:
We'll need to update the config files in use_cases and tests/use_cases as the old draw nodes are deprecated, but lets do this right before releasing another version with these UI changes to avoid confusion for current users